src/main/java/core/mate/academy/service/TruckProducer.java

```diff
@@ -0,0 +1,24 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Machine;
+import core.mate.academy.model.Truck;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer<Machine> {
+    private List<Machine> truckList = new ArrayList<>();
+
+    public TruckProducer() {
+        truckList.add(new Truck());
+        truckList.add(new Truck());
+        truckList.add(new Truck());
+    }
+
+    @Override
+    public List<Machine> get() {
+        if (truckList.isEmpty()) {
+            return null;
+        }
+        return truckList;

```

Comment:
create instances and return them here directly - we can even do it in one line via List.of(). Fix for each producer